### PR TITLE
abstracted out the conversation title functions into their own class s…

### PR DESCRIPTION
**Pull request by Dan2anderson**
…o that we can subclass it and easily override those methods as needed.

We changed our implementation to match the way you requested in this pull request (https://github.com/layerhq/Atlas-Android/pull/71/files) we closed the old pull request.

### CODE REVIEW

File: layer-atlas/src/main/java/com/layer/atlas/util/ConversationFormatter.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2017 Sales Rabbit, Inc. All rights reserved.
```
Comment (andela-wakinsanmi) [inline layer-atlas/src/main/java/com/layer/atlas/util/ConversationFormatter.java]: @Dan2anderson Thank you so much for the pr. Can you please remove the copyright?

Thanks

**Comment by Dan2anderson** [comment]
ha sorry about that.  it's been removed.

File: layer-atlas/src/main/java/com/layer/atlas/util/ConversationFormatter.java (hunk 1 above)
Comment (Dan2anderson) [inline layer-atlas/src/main/java/com/layer/atlas/util/ConversationFormatter.java]: done

Comment (andela-wakinsanmi) [inline layer-atlas/src/main/java/com/layer/atlas/util/ConversationFormatter.java]: Thanks @Dan2anderson

**Pull request merged**